sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -114,15 +114,15 @@ class DateTime final : public _detail::Clock::time_point {
   }
 
   /**
-   * @brief Construct an instance of #Azure::DateTime from base class.
+   * @brief Constructs an instance of `DateTime` from a `time_point`.
    *
    */
   constexpr DateTime(time_point const& timePoint) : time_point(timePoint) {}
 
   /**
-   * @brief Construct an instance of #Azure::DateTime from
-   * `std::chrono::system_clock::time_point`.
+   * @brief Construct an instance of `%DateTime` from `std::chrono::system_clock::time_point`.
    * @param systemTime A value of `std::chrono::system_clock::time_point`.
+   *
```

Comment:
@vhvb1989 is this extra new line required still?

Reply:
Maybe will be in a follow up PR, depending on the answer

Reply:
Victor "We want it for docs inside a class, like properties and methods.

We don't want it for documentation out of classes, like struct/class/namespace/global stuff"